src/printer.js

```diff
@@ -65,12 +64,14 @@ Printer.prototype.print = function(callback) {
 	async.eachSeries(
 		_self.commandQueue,
 		function(command, callback) {
-			if (_self.commandDelay !== 0) {
-				sleep.usleep(_self.commandDelay);
+			function write() {
+				_self.serialPort.write(command, function() {
+					_self.serialPort.drain(callback);
+				});
 			}
-			_self.serialPort.write(command, function() {
-				_self.serialPort.drain(callback);
-			});
+
+			if (_self.commandDelay === 0) return write()
```

Comment:
this would be sufficient since if the commandDelay is 0 then the timeout will be 0 too.
```
setTimeout(write, Math.ceil(_self.commandDelay / 1000));
```